plugins/faqs/enhance.js

```diff
@@ -0,0 +1,41 @@
+function slugify(input) {
+  return String(input)
+    .trim()
+    .toLowerCase()
+    .replace(/([^\w]|\s)+/g, '-')
+    .replace(/-$/g, '')
+}
+
+export default ({ Vue, siteData }) => {
+  Vue.mixin({ methods: { slugify } })
+
+  const faqPage = siteData.pages
+    .find(page => page.path === '/guide/learning/faq.html')
+
+  if ( ! faqPage) {
+    throw new Error("The FAQ plugin couldn't find the FAQ page (guide/learning/faq.md)! Did its path change?")
+  }
+
+  faqPage.headers = faqPage.faqData
+    .reduce((headers, faq) => {
+      const category = faq.category || 'Uncategorised'
```

Comment:
This is the second time we fallback to 'Uncategorised'. Do we need to do it in two places?

Reply:
Definitely not, agreed.